pom.xml

```diff
@@ -236,6 +236,11 @@
         <artifactId>plexus-interpolation</artifactId>
         <version>1.19</version>
       </dependency>
+      <dependency>
```

Comment:
Where is this one used? I was trying to remove the dependency on it from the configuration modules, so I'd like to know if there are just lingering, unused imports somewhere.

Reply:
The propulsor-configuration-dotconf module has this dependency. Not sure if it is still used. I'll try to remove it to see if any compile errors.